polaris-react/src/components/Columns/Columns.scss

```
@@ -22,6 +22,8 @@
   display: grid;
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   grid-template-columns: var(--pc-columns-grid-template-columns-xs);
+  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+  align-items: var(--pc-columns-align-items);
```

Do we need to setup this custom property with `initial`? Otherwise, nested `Columns` layouts will inherit the parents `alignItems` prop, right?

---

If the custom property isn't set, it fails silently to the browser default of `normal`. It doesn't inherit the parent `align-items` value.

I can add `normal` as the default if the prop isn't set to make it explicit.

---

Here's a demo:
https://codepen.io/aaronccasanova/pen/rNZzJrz

---

your demo uses `justify-content` but this is for `align-items`. If you update the demo to `align-items` it seems to behave as expected?